### ISSUE TITLE
event change:date - is it working ?

### ISSUE THREAD

**Issue by natanraj** [eclipsesource/tabris-calendar#8]
Unable to make it work even from the example. I am using a Redmi note 3

**Comment by cookieguru** [comment]
As of 3182edc57c872078a8f624e116c9d34b9bb92248 this plugin uses the same camelCased event names as all other events do in Tabris.js 2.x.  The readme hasn't been updated but [the example](https://github.com/eclipsesource/tabris-plugin-calendar/blob/master/example/src/app.js) should work.

**Comment by natanraj** [comment]
ok . it is triggering now, but throws maximum call stack range reached error.
calendarContainer is my composite where i have an button extra


`let cldnr = new escalendar.Calendar({
          left: 0, right: 0, top: 0, bottom: 0,
          maxDate: (new Date()).getTime() // year 2000
        }).appendTo(calendarContainer);
      cldnr.on('dateChanged', function(calendarEvnt){
          console.log(calendarEvnt);
      })`

**Comment by natanraj** [comment]
i am testing this through server mode 'tabris serve'. Not sure if it has any implications

**Comment by natanraj** [comment]
i tried in a built app. It is working fine.

**Comment by cpetrov** [comment]
@natanraj Thank you for opening this issue and sorry for the inconvenience. I've published a version 1.0.1 of the plugin with an updated README. And thanks @cookieguru for the fast PR!

@natanraj I could not reproduce your second problem using the following snippet:
```js
const {ui} = require('tabris');

let cldnr = new escalendar.Calendar({
  left: 0, right: 0, top: 0, bottom: 0, maxDate: (new Date()).getTime()
}).appendTo(ui.contentView);
cldnr.on('dateChanged', function(calendarEvnt) {
  console.log(calendarEvnt);
})
```
Please open a new issue if the app is still crashing for you when served using `tabris serve`.
